src/parser/transform/statement/transform_alter_table.cpp

```diff
@@ -34,13 +35,18 @@ unique_ptr<AlterStatement> Transformer::TransformAlter(duckdb_libpgquery::PGNode
 			}
 			auto centry = TransformColumnDefinition(cdef);
 
+			if (centry.IsSerial()) {
+				throw ParserException("Adding columns with constraints not yet supported");
```

Comment:
small nit: I think NotImplemented would be a better type, as it does not fail to parse, it's just not implemented

Reply:
thanks for the suggestion!